bcda/client/bluebutton.go

```diff
@@ -187,17 +187,16 @@ func logRequest(req *http.Request, resp *http.Response, jobID string) {
 		"bb_query_ts": req.Header.Get("BlueButton-OriginalQueryTimestamp"),
 		"bb_uri":      req.Header.Get("BlueButton-OriginalUrl"),
 		"job_id":      jobID,
-	}).Infoln("Blue Button request")
+	}).Infoln("request")
 
 	if resp != nil {
 		logger.WithFields(logrus.Fields{
 			"resp_code":      resp.StatusCode,
-			"bb_query_id":    resp.Header.Get("BlueButton-OriginalQueryId"),
-			"bb_query_ts":    resp.Header.Get("BlueButton-OriginalQueryTimestamp"),
-			"bb_uri":         resp.Header.Get("BlueButton-OriginalUrl"),
+			"bb_query_id":    req.Header.Get("BlueButton-OriginalQueryId"),
```

Comment:
shouldn't these values be set from the resp param and not the req??

Reply:
Those headers used to get populated in the BB responses I think (which is why we had it set up that way originally) but don't anymore. So request is the only place where we can get them.

Reply:
oh okay, so BB changed the way they pass the data back to us. Gotcha